File: src/uk/dwelsh/kaffee/models/RfidTag.java
package uk.dwelsh.kaffee.models;

public class RfidTag {
    private String userId;
    private String tag;
    private Coffee coffee;

    public RfidTag(String userId, String tag, Coffee coffee) {
        this.userId = userId;
        this.tag = tag;
        this.coffee = coffee;
    }

    public String getUserId() {
        return userId;
    }

    public String getTag() {
        return tag;
    }

    public Coffee getCoffee() {
        return coffee;
    }

    @Override
    public String toString() {
        return "RfidTag{" +
                "userId='" + userId + '\'' +
                ", tag='" + tag + '\'' +
                ", coffee=" + coffee +
                '}';
    }
}

File: src/main/kotlin/uk/dwelsh/kaffee/models/RfidTag.kt
package uk.dwelsh.kaffee.models

class RfidTag(val userId: String, val tag: String, val coffee: Coffee) {

    override fun toString(): String {
        return "RfidTag{" +
                "userId='" + userId + '\'' +
                ", tag='" + tag + '\'' +
                ", coffee=" + coffee +
                '}'
    }

}